manim/mobject/geometry/line.py

```diff
@@ -1015,6 +1015,38 @@ def construct(self):
             return self.angle_value / DEGREES
         return self.angle_value
 
+    @staticmethod
+    def from_three_points(
+        A: np.ndarray, B: np.ndarray, C: np.ndarray, **kwargs
+    ) -> Angle:
+        """The angle between the lines AB and BC.
+
+        This constructs the angle :math:`\angle ABC`.
+
+        Parameters
+        ----------
+        A: The endpoint of the first angle leg
+        B: The vertex of the angle
+        C: The endpoint of the second angle leg
+
+        **kwargs
+            Further keyword arguments are passed to :class:`.Angle`
+
+        Returns
+        -------
+        The Angle calculated from the three points
+
+        Examples
+        --------
+        .. manim:: AngleWithThreePointsExample
+            :save_last_frame:
+            >>> A = np.array(UP)
+            >>> B = np.array(ORIGIN)
+            >>> C = np.array(RIGHT)
+            >>> ang = Angle.from_three_points(A, B, C)
```

Comment:
```suggestion
        .. manim:: AngleWithThreePointsExample
            :save_last_frame:

            class AngleWithThreePointsExample(Scene):
                def construct(self):
                    right_angle = Angle.from_three_points(UP, ORIGIN, RIGHT)
                    self.add(right_angle)
```

Formatting was not correct, no example was actually rendered. A basic example like this is fine, but it might also make sense to include at least a second angle where passing keyword arguments to `Angle` (like `other_angle=True`, for example) is illustrated.

Reply:
I added an example! What is the best way to visualize the examples?